.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.9.7","bspVersion":"2.1.0-M1","languages":["scala"],"argv":["/Users/omotayotalabi/Library/Java/JavaVirtualMachines/corretto-17.0.8/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/omotayotalabi/Library/Application Support/JetBrains/IdeaIC2022.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/omotayotalabi/Library/Application%20Support/JetBrains/IdeaIC2022.1/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
think we don't need this folder .bsp, can you remove please and also add .bsp in gitignore